projects/dsp-ui/src/lib/action/components/confirmation-dialog/confirmation-dialog.component.spec.ts

```diff
@@ -99,10 +110,19 @@ describe('ConfirmationDialogComponent', () => {
             expect(dialogDiv).toBeTruthy();
```

Comment:
Couldn't you use the harnesses also in this `it`? I think then you could get rid of the `whenStable`.

Reply:
If I use a harness here, I don't know how to select a basic `<p>` element to check if the title is displayed correctly